katas/langtonant/richard/langtonant.test.js

```diff
@@ -0,0 +1,158 @@
+import { isSquareBlack } from './verifiers.js';
+import { isSquareLimit } from './verifiers.js';
+import { changePositionX } from './changers.js';
+import { changePositionY } from './changers.js';
+import { ChangeColor } from './changers.js';
+import { ChangeDirection } from './changers.js';
+//import { printLimit } from './printers.js';
+//import { printGrid } from './printers.js';
```

Comment:
I would recommend try to remove commented code, they tend to add distraction to readability